http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200609_add_persistent_disk_formattedBy.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="gabriela" id="add_persistent_disk_formattedBy">
+        <addColumn tableName="PERSISTENT_DISK">
+            <column name="formattedBy" type="VARCHAR(255)" defaultValue="NULL">
```

Comment:
I think we were saying making this a ENUM type, similar to https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_cluster_table_gce_refactor.xml#L21

Reply:
+1 on using enum

Reply:
will update

Reply:
`NULL` is an invalid default value for ENUM so looks like we should keep `VARCHAR(255)` since `formattedBy` is an optional field. All other enums use varchar except the one in the link above that Qi mentioned

Reply:
NULL should be valid, see https://www.tutorialspoint.com/what-are-the-different-attributes-of-MySQL-ENUM-data-type#:~:text=NOT%20NULL%20%2D%20In%20ENUM%20type,while%20describing%20the%20ENUM%20column...



I tried

```
        <addColumn tableName="PERSISTENT_DISK">
            <column name="formattedBy" type="ENUM('GALAXY', 'GCE')">
                <constraints nullable="true"/>
            </column>
        </addColumn>
```
this should work